pocs/focuser/birger.py

```diff
@@ -112,7 +130,6 @@ def move_to(self, position):
         Does not do any checking of the requested position but will warn if the lens reports hitting a stop.
         Returns the actual position moved to in lens encoder units.
         """
-        self.logger.debug('Moving focus to {} encoder units'.format(position))
```

Comment:
I've certainly kept use of `INFO` to a minimum in the `Camera` and `Focuser` code. Basically a message or two on initialisation, plus one at the end of an autofocus if plotting saving where it was written to.